common/src/main/java/com/microsoft/identity/common/internal/platform/DevicePopManager.java

```diff
@@ -799,16 +801,16 @@ String getPublicKey(@NonNull final PublicKeyFormat format) throws ClientExceptio
     @Override
     public Certificate[] getCertificateChain() throws ClientException {
         return mKeyManager.getCertificateChain();
-   }
+    }
 
     private @NonNull
     String getJwkPublicKey() throws ClientException {
         final Exception exception;
         final String errCode;
 
         try {
-            final net.minidev.json.JSONObject jwkJson = getDevicePopJwkMinifiedJson();
-            return jwkJson.getAsString(SignedHttpRequestJwtClaims.JWK);
+            final Map<String, Object> jwkMap = getDevicePopJwkMinifiedJson();
```

Comment:
Why making this change in this PR?

Reply:
This is a goof... for some reason that code as it was wasn't building so I took the latest from dev...  couldn't resolve a dependency that was subequently removed.  I"ll bring back the old implementation.